QRCoderConsole/Properties/AssemblyInfo.cs

```diff
@@ -1,27 +0,0 @@
-﻿using System.Reflection;
```

Comment:
Version is moved into csproj file. I am not sure about the values there.

Reply:
But what's about the other properties like "Description", "Product", ... - I think at least for the compile targets net35, net40 the AssemblyInfo.cs is still used. Isn't it? So I would say it doesn't hurt to keep the AssemblyInfo or do I oversee something?

Reply:
I guess since we use the new project style here, it will anyway not loadable in VS2015 and before. The entries in csproj is a transformation to Assemblyinfo.cs. What I do not understand for what you need support for these ancient versions. Since the oldest Windows 10, is I guess LTSB from 2016, which if this page fits https://docs.microsoft.com/de-de/dotnet/framework/migration-guide/versions-and-dependencies 4.6.2. This means basically nobody will have need to use older version if he wants security. But sure we can leave it with a flag.